Sources/XMTPiOS/Contacts.swift

```diff
@@ -55,12 +55,13 @@ public class ConsentList {
 		identifier = try? LibXMTP.generatePrivatePreferencesTopicIdentifier(privateKey: privateKey)
 	}
 
-	func load() async throws -> ConsentList {
+	func load(pagination: Pagination? = nil) async throws -> ConsentList {
 		guard let identifier = identifier else {
 			throw ContactError.invalidIdentifier
 		}
 
-		let envelopes = try await client.apiClient.envelopes(topic: Topic.preferenceList(identifier).description, pagination: Pagination(direction: .ascending))
+		let pagination = pagination ?? Pagination(direction: .ascending)
```

Comment:
I think we have to force whatever they pass to have the direction of ascending.... Otherwise the list will load backwards and it would be the opposite of what the actual consent is. We probably need to protect against that.